src/etc/inc/certs.inc

```diff
@@ -281,6 +281,42 @@ function ca_inter_create(& $ca, $keylen, $lifetime, $dn, $caref, $digest_alg = "
 	return true;
 }
 
+function ca_trustedcerts_generate() {
+	global $config, $g;
+	if (isset($config['system']['developerspew'])) {
+		$mt = microtime();
+		echo "ca_trustedcerts_generate() being called $mt\n";
+	}
+
+	mwexec("/bin/mkdir -p {$g['etc_path']}/ssl/certs");
+	/* Remove all certs in the directory (but leave symlinks in place to allow power users to add trusted certs outside the Certificate Manager) */
+	mwexec("/usr/bin/find {$g['etc_path']}/ssl/certs -type f -exec /bin/rm '{}' ';'");
```

Comment:
I agree with this and the previous comment that I should be using pfSense library functions over shell commands. The `mkdir` code is easy to change; I had aped other code in `certs.inc` that did the same thing, so my next pull request may be a few patches to clean up that file.

I will refactor to eliminate the call to `/usr/bin/find` and use `is_link` and `unlink` instead. There's no way to use prefixes in `/etc/ssl/certs`, unfortunately, because OpenSSL requires that the files be named `{cert_hash}.0`, and OpenSSL doesn't support certificates in subdirectories of cert_path, so I can't do that either. Symlinks seemed a reasonable compromise to not completely obliterate the ability of power users and developers to manually place certificates in the directory.